nlprule/src/compile/mod.rs

```diff
@@ -81,6 +89,8 @@ pub enum Error {
     Unimplemented(String),
     #[error("error parsing to integer: {0}")]
     ParseError(#[from] ParseIntError),
+    #[error("nlprule error: {0}")]
```

Comment:
With `#[from]` adding a explicit print `{0}` would duplicate the error, since the backtrace will also contain the `cause` which is implicit by `#[from]`.

So `#[error(transparent)]` or `#[error("nlprule error")]`.